sound/sound-font-config.ini-configuration.md

```diff
@@ -25,111 +27,206 @@ volHum=15
 
-# Only used for non-smoothswing fonts. Specifies how aggressive a swing has to be to be considered a slash. Once we
+# Only used for non-smoothswing fonts. Specifies how aggressive a swing has
+# to be to be considered a slash. Once we
 # reach the ProffieOSSwingSpeedThreshold, rate of swing speed change is used to
-# determine if it's a swing or a slash. Default 260
+# determine if it's a swing or a slash.
+# Default 260
 ProffieOSSlashAccelerationThreshold=260
 
-# For OLED displays. This specifies the frame rate of animations.
+# If #define ENABLE_SPINS is defined:
+# Number of degrees the blade must travel while staying above the
+# swing threshold in order to trigger a spin sound.
+# Default is 360 or one full rotation.
+ProffieOSSpinDegrees=360.0
+
+# ---- As of ProffieOS 6 ---- #
+
+# Minimum acceleration for Accent Swing file Selection
+# recommended value is 20.0 ~ 30.0
+ProffieOSMinSwingAcceleration=20.0
+
+# Maximum acceleration for Accent Swing file Selection
+# must be higher than Min value to enable selection
+# recommended value is 100.0 ~ 150.0
+ProffieOSMaxSwingAcceleration=100.0
+
+# Set the volume for each effect individually, in percent.
+# 50 makes it half as loud. 200 makes it twice as loud. 
+# Maximum allowed value is currently 255. The default is 100.
+# EFFECTNAME can be "clash", “preon”, “out”, “pstoff”, etc.
+#
+# ProffieOS.SFX.EFFECTNAME.volume=100
+ProffieOS.SFX.boot.volume=75
+
+# Make smoothswings start in sync with hum.
+# Set to 1 to sync, or 0 to resume swings where last pair left off.
+ProffieOSSmoothSwingHumstart=1
+
+# Pair sequential effects such as preon->out.wav files.
+# EFFECTNAME can be “preon”, “out”, "lock", “in”, etc.
+# Set to 1 *AND* have the same number of files in the second effect
+# to activate this feature. Only the first in the chain need be paired here.
+# ProffieOS.SFX.EFFECTNAME.paired=1
+# For example:
+# pair preon -> out.wavs
+#ProffieOS.SFX.preon.paired=1
+# pair in -> pstoff.wavs
+#ProffieOS.SFX.in.paired=1
+
+# ---- As of ProffieOS 7 ---- #
+# bgnlock -> lock
+#ProffieOS.SFX.bgnlock.paired=1
+# lock -> endlock
+#ProffieOS.SFX.lock.paired=1
+#ProffieOS.SFX.bgndrag.paired=1
+#ProffieOS.SFX.drag.paired=1
+#ProffieOS.SFX.bgnlb.paired=1
+#ProffieOS.SFX.lb.paired=1
+#ProffieOS.SFX.bgnmelt.paired=1
+#ProffieOS.SFX.melt.paired=1
+#ProffieOS.SFX.bgnarm.paired=1
+# armhum -> endarm (if disarmed) -> boom
+#ProffieOS.SFX.armhum.paired=1
+
+# out -> hum
+#ProffieOS.SFX.out.paired=1
+
+# hum -> in
+# *Note* When multiple hum files exist,
+# even if not the same number of files as in.wavs,
+# (so it won't pair to them,) this also will loop the 
+# same hum sound until next ignition instead of randomly 
+# choosing a different file once the current hum ends,
+#ProffieOS.SFX.hum.paired=1
+
+# Set smoothswings to start in sync with hum start.
+# Set to 1 to sync, or 0 to resume swings where last pair left off.
+# Default 0
+ProffieOSSmoothSwingHumstart=1
+
+# ---- OLED displays ---- #
```

Comment:
So, are all the variables below here OS7 only?
Are we still under the "As of OS7" heading, or do we start over here? It's not very clear.

Reply:
fixed. How about now?